pages/common/pio-test.md

```diff
@@ -0,0 +1,28 @@
+# pio test
+
+> Run local tests on a PlatformIO project.
+> More information: <https://docs.platformio.org/en/latest/core/userguide/cmd_test.html>.
+
+- Run all tests in all environments of the current PlatformIO project:
+
+`pio test`
+
+- Test only specific environments:
+
+`pio test --environment {{environment1}} --environment {{environment2}}`
+
+- Run only tests whose name matches a specific pattern:
+
+`pio test --filter "{{pattern}}"`
```

Comment:
I think you missed my other comment - is this a regex, or something else?

Reply:
Something specific. It just supports a few [basic options](https://docs.platformio.org/en/latest/core/userguide/cmd_test.html#cmdoption-pio-test-f).

Reply:
That'd be a glob pattern.